web/modules/custom/feedback/feedback.routing.yml

```diff
@@ -1,41 +0,0 @@
-feedback.content:
-  path: '/feedback'
-  defaults:
-    _controller: 'Drupal\feedback\Controller\FeedbackController::content'
-    _title: 'Feedback'
-  requirements:
-    _permission: 'access content'
-
-feedback.admin_settings:
-  path: '/admin/config/content/feedback'
-  defaults:
-    _form: '\Drupal\feedback\Form\FeedbackSettingsForm'
-    _title: 'Feedback Settings'
-  requirements:
-    _permission: 'administer feedback'
-
-feedback.report:
-  path: '/admin/reports/feedback'
-  defaults:
-    _controller: '\Drupal\feedback\Controller\ReportController::description'
-    _title: 'List of Feedback'
-  requirements:
-    _permission: 'administer feedback'
-
-feedback.delete_form:
-  path: '/admin/reports/feedback/delete/{cid}'
```

Comment:
path: '/admin/reports/feedback/delete/{cid}

why "{cid}". short version of "feedback id"  will be "fid"